resources/wikia/modules/iframeWriter.js

```diff
@@ -17,6 +17,7 @@ define('wikia.iframeWriter', [
 	 * @param {string} params.code - code to put into the iframe
 	 * @param {number} params.width - desired width of the iframe
 	 * @param {number} params.height - desired height of the iframe
+	 * @param {array} [params.classes] - custom classes of the iframe
```

Comment:
Let's have the {string} params.class param and just assign it to iframe.class when creating iframe.